spec/features/applications/show_spec.rb

```diff
@@ -96,4 +96,39 @@
       end
     end
   end
+
+  #As a visitor
+  # When I visit an application's show page
+  # And I search for a Pet by name
+  # And I see the names Pets that match my search
+  # Then next to each Pet's name I see a button to "Adopt this Pet"
+  # When I click one of these buttons
+  # Then I am taken back to the application show page
+  # And I see the Pet I want to adopt listed on this application
+
+  describe "User story 5" do
+    describe "Search a pet by name" do
+      it "has a button 'Adopt this Pet' next to the pet's name" do
+        @pet_1 = @application_1.pets.create!(
```

Comment:
I think for this test setup we need to have this just be `pet_1 = Pet.create!(` so the pet isn't already connected to the application